VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/private-consultation-room-controls/private-consultation-room-controls.component.scss

```diff
@@ -136,51 +137,49 @@
 
 .dropdown {
   position: relative;
-  display: none;
-  @media only screen and (min-width: 320px) and (max-width: 768px) {
-    display: inline-block;
+  display: inline-block;
+  @media only screen and (min-width: $breakpoint-large) {
+    display: none;
   }
 }
 
 .dropdown-content {
   display: block;
   position: absolute;
-  right: -73px;
+  right: -6.25rem;
   left: auto;
   background-color: govuk-colour('white');
   color: black;
-  font-size: 16px;
-  min-width: 300px;
+  font-size: rem(16px);
+  width: 12.5rem;
   box-shadow: 0px 8px 16px 0px rgba(0, 0, 0, 0.2);
   padding: 12px 16px;
   z-index: 999;
 }
+
 .context-item {
   font-family: 'GDS Transport', arial, sans-serif;
   -webkit-font-smoothing: antialiased;
   margin-left: 4px;
 }
-.portrait {
+.hide-for-medium {
   display: block;
-  @media only screen and (orientation: landscape) {
+  @media only screen and (min-width: $breakpoint-medium) {
     display: none;
   }
 }
 
-.landscape {
-  display: inline-flex;
-  @media only screen and (orientation: portrait) {
-    display: none;
-  }
-  @media only screen and (min-width: 768px) {
-    display: none;
+.show-for-medium {
+  display: none;
+  @media only screen and (min-width: $breakpoint-medium) and (max-width: $breakpoint-large) {
```

Comment:
```suggestion
  @media only screen and (min-width: $breakpoint-medium) {
```

Reply:
we only wnat show the button (min-width: $breakpoint-medium) and (max-width: $breakpoint-large)

Reply:
Oh? I didn't realise there was more than the 2 sets of controls. I'm not really clear why we would need that. I would need to see it in action. I feel like this isn't the best way to do it but I would need to see it in action. Could you change the class to show-for-medium-only

However this will probably display both controls on 1024px wide, could you check this? The max width would probably need to be 1px less than $breakpoint-large.

Reply:
I add new $breakpoint-medium-max with that 1023px.